File: redcap_v15.7.6/API/examples/r/delete_user_roles.R
#!/usr/bin/env Rscript

source('config.R')
library(RCurl)

result <- postForm(
    api_url,
    token=api_token,
    content='userRole',
    action='delete',
    'roles[0]'='U-522RX7WM49'
)
print(result)

File: html/redcap_v15.7.6/API/examples/ruby/delete_user_roles.rb
#!/usr/bin/env ruby

require 'json'
require 'curl'
require './settings.rb'
include Settings

fields = {
  :token => Settings::API_TOKEN,
  :content => 'userRole',
  :action => 'delete',
  :format => 'json',
  'roles[0]' => 'U-522RX7WM49',
}

ch = Curl::Easy.http_post(
  Settings::API_URL,
  fields.collect{|k, v| Curl::PostField.content(k.to_s, v)}
)
puts ch.body_str
